cookbook/examples/apps/ai_tutor/db.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime
+
+from dotenv import load_dotenv
+from pymongo import MongoClient
+
+load_dotenv()
+
+MONGO_URI = "mongodb://localhost:27017/"
+client = MongoClient(MONGO_URI)
```

Comment:
@Itz-Antaripa I'm a bit confused. Even if i dont run the Mongo Server locally it works?
Where is it storing things? 

Maybe thats why when i click on generate quiz it fails with this-
![image](https://github.com/user-attachments/assets/064fcc78-06bf-4b69-baf0-f877e2d85d1a)

Can you check the db part again? Also the MONGO_URI is hardcoded here and above `load_dotenv()` is used which makes it redundant then.

If db is not connected, we shouldn't show this-
![image](https://github.com/user-attachments/assets/84dd95b1-e390-4981-89d0-3d442e6d445b)